src/Xamarin.Android.Build.Tasks/Tasks/CheckForInvalidDesignerConfig.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.Android.Build.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.IO;
+
+namespace Xamarin.Android.Tasks
+{
+	/// <summary>
+	/// Emits XA1034 is an assembly has a reference to _Microsoft.Android.Resource.Designer.dll
+	/// * NOTE: only called when $(AndroidUseDesignerAssembly) is false
+	/// </summary>
+	public class CheckForInvalidDesignerConfig : AndroidTask
```

Comment:
Do we have any timings on who quick this tasks is ? Have we not just moved the 200 ms to this task instead? 
I think this is now working on a smaller subset of assemblies , but it would be good to know how quick this new task is.

Reply:
This task is the exact same speed, but I will add a check to skip BCL assemblies might make it 50% faster.

Reply:
Ok, the new code is actually faster...

![image](https://user-images.githubusercontent.com/840039/211612435-6e8df67b-97e5-4273-8dff-0e0f3943d3cd.png)

It's because the task previously looked at *all* assemblies, and now this only looks at `@(_MonoAndroidReferencePath);@(_MonoAndroidReferenceDependencyPaths)`.